### ISSUE TITLE
"pred_wh not finite" 

### ISSUE THREAD

**Issue by ouyangpingbu** [duwt/OrienMask#6]
In train phase,  why does "pred_wh not finite" appear and how to fix it ?? thanks.
2021-08-13 11:54:59,609 [EPOCH 1]
2021-08-13 11:54:59,609 Train on epoch 1
  0%|▍                                                                                                                       | 1/313 [00:05<28:19,  5.45s/it, loss=-1.0000, lr=-1.00e0]2021-08-13 11:55:05,191 Reducer buckets have been rebuilt in this iteration.
 92%|██████████████████████████████████████████████████████████████████████████████████████████████████████████▏        | 289/313 [02:44<00:13,  1.76it/s, lr=5.50e-04, loss=1257.3200]pred_wh not finite
pred_wh not finite
 92%|██████████████████████████████████████████████████████████████████████████████████████████████████████████▏        | 289/313 [02:44<00:13,  1.76it/s, lr=5.50e-04, loss=1257.3200]
INFO:torch.distributed.elastic.agent.server.api:[default] worker group successfully finished. Waiting 300 seconds for other agents to finish.

**Comment by duwt** [comment]
Increasing the learning rate warm-up steps from 500 to 1000 can upgrade the stability. Refer to the new commit https://github.com/duwt/OrienMask/commit/61a0aa8f6b34c699962f83bb6c8ce557337ea701 for more details.
